components/automate-cli/pkg/adminmgmt/adminmgmt.go

```diff
@@ -8,15 +8,9 @@ import (
 	grpc_status "google.golang.org/grpc/status"
 
 	authz_constants "github.com/chef/automate/components/authz-service/constants"
-	authz_constants_v1 "github.com/chef/automate/components/authz-service/constants/v1"
 	authz_constants_v2 "github.com/chef/automate/components/authz-service/constants/v2"
 	"github.com/chef/automate/components/automate-cli/pkg/client"
 	"github.com/chef/automate/components/automate-cli/pkg/status"
-	teams "github.com/chef/automate/components/automate-gateway/api/auth/teams"
-	teams_req "github.com/chef/automate/components/automate-gateway/api/auth/teams/request"
-	users_req "github.com/chef/automate/components/automate-gateway/api/auth/users/request"
-	"github.com/chef/automate/components/automate-gateway/api/authz"
-	authz_req "github.com/chef/automate/components/automate-gateway/api/authz/request"
 	authz_v2_req "github.com/chef/automate/components/automate-gateway/api/iam/v2/request"
```

Comment:
Per previous team discussion, let's try to minimize v2 references where possible. So this should become:
```suggestion
	authz_req "github.com/chef/automate/components/automate-gateway/api/iam/v2/request"
```